project_task_stage_allow_timesheet/views/account_analytic_line.xml

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright 2018 ACSONE SA/NV
+     License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl). -->
+
+<odoo>
+
+    <record model="ir.ui.view" id="hr_timesheet_line_form">
```

Comment:
just cosmetical change (optional):
id before model
and use of original id (hr_timesheet_line_form)